### ISSUE TITLE
Support different icon/emoji in alerting title depending on the kind of message + title templating

### ISSUE THREAD

**Issue by Byh0ki** [TwiN/gatus#927]
### Describe the feature request

Hello,

First of all, thank you for the app, it's very good!

I would be interested in having a way to modify the icon/emoji ( :rescue_worker_helmet: ) in the notifications titles based on the notification status (either firing or resolved). We might also want to allow modifying the title template to include the target (and maybe more?). 

Related to https://github.com/TwiN/gatus/issues/508 but maybe it could be standardized for all notification methods.

### Why do you personally want this feature to be implemented?

On slack mobile, only the title is displayed in the app notification and being able to quickly see if a previous alert has been resolved without opening the app would be quite nice!

![android-slack-notif](https://github.com/user-attachments/assets/6a04b5fc-6692-4cd0-9744-0a7a164551aa)

### How long have you been using this project?

~3y

**Comment by Byh0ki** [comment]
Hello @TwiN, are we able to override the title from an endpoint alert?

To come back to the original need, I'm not sure we can close this issue as the new feature doesn't allow us to have a title for alerts and resolution if I understand correctly the doc :/

**Comment by TwiN** [comment]
#1079 added support for doing that with Slack alerts. It's not a global configuration at the moment.

**Comment by mverkleij-dlx** [comment]
You can override the title for individual alerts using provider alerts like so:
```yaml
endpoints:
  - name: dynatron.nl 
    alerts:
      - type: slack
        provider-override:
          title: dynatron.nl DOWN
```

**Comment by laurentlbm** [comment]
I agree with the original request, it would be great to be able to customize the title based on the status (triggered or resolved). I'm using discord notifications at the moment and I would love to be able to see that an error has already been resolved with just a quick glance at the notification titles.
